app/javascript/components/property-mapping-list/PropertiesList.jsx

```diff
@@ -219,6 +227,19 @@ export default class PropertiesList extends Component {
     });
   };
 
+  /**
+   * Set the alignment score from the calculation when its done
+   *
+   * @param {Object} property
+   * @param {Float} score
+   */
+  handleAlignmentScoreFetched = (property, score) => {
+    const { properties } = this.state;
+    property.alignmentScore = score;
+
+    this.setState({ properties: [...properties] });
```

Comment:
`properties` is an array already, no need to re-expand it. I believe, it can be shortened to `this.setState({ properties })`.